docs/modules/ROOT/pages/spring-cloud-commons/common-abstractions.adoc

```diff
@@ -204,11 +331,55 @@ public class MyClass {
 The URI needs to use a virtual host name (that is, a service name, not a host name).
 The Spring Cloud LoadBalancer is used to create a full physical address.
 
-IMPORTANT: If you want to use a `@LoadBalanced WebClient.Builder`, you need to have a load balancer
+IMPORTANT: If you want to use a `@LoadBalanced WebClient.Builder`, you need to have a Spring Cloud LoadBalancer
 implementation in the classpath. We recommend that you add the
 xref:spring-cloud-commons/loadbalancer.adoc#spring-cloud-loadbalancer-starter[Spring Cloud LoadBalancer starter] to your project.
 Then, `ReactiveLoadBalancer` is used underneath.
 
+[[multiple-webclient-objects]]
+=== Multiple `WebClient.Builder` Objects
```

Comment:
Remove the ticks around `WebClient.Builder`. The code font does not render correctly in headings.